src/aura/AuraAvatarTwoToken.sol

```diff
@@ -0,0 +1,755 @@
+        }
+
+        uint256 oldMinOutBpsBalToUsdcVal = minOutBpsBalToUsdcPtr.val;
+        minOutBpsBalToUsdcPtr.val = _minOutBpsBalToUsdcVal;
+
+        emit MinOutBpsBalToUsdcValUpdated(_minOutBpsBalToUsdcVal, oldMinOutBpsBalToUsdcVal);
+    }
+
+    function setMinOutBpsAuraToUsdcVal(uint256 _minOutBpsAuraToUsdcVal) external onlyOwnerOrManager {
+        if (_minOutBpsAuraToUsdcVal > MAX_BPS) {
+            revert InvalidBps(_minOutBpsAuraToUsdcVal);
+        }
+
+        BpsConfig storage minOutBpsAuraToUsdcPtr = minOutBpsAuraToUsdc;
+
+        uint256 minOutBpsAuraToUsdcMin = minOutBpsAuraToUsdcPtr.min;
+        if (_minOutBpsAuraToUsdcVal < minOutBpsAuraToUsdcMin) {
+            revert LessThanBpsMin(_minOutBpsAuraToUsdcVal, minOutBpsAuraToUsdcMin);
+        }
+
+        uint256 oldMinOutBpsAuraToUsdcVal = minOutBpsAuraToUsdcPtr.val;
+        minOutBpsAuraToUsdcPtr.val = _minOutBpsAuraToUsdcVal;
+
+        emit MinOutBpsAuraToUsdcValUpdated(_minOutBpsAuraToUsdcVal, oldMinOutBpsAuraToUsdcVal);
+    }
+
+    function setMinOutBpsBalToBptVal(uint256 _minOutBpsBalToBptVal) external onlyOwnerOrManager {
+        if (_minOutBpsBalToBptVal > MAX_BPS) {
+            revert InvalidBps(_minOutBpsBalToBptVal);
+        }
+
+        BpsConfig storage minOutBpsBalToBptPtr = minOutBpsBalToBpt;
+
+        uint256 minOutBpsBalToBptMin = minOutBpsBalToBptPtr.min;
+        if (_minOutBpsBalToBptVal < minOutBpsBalToBptMin) {
+            revert LessThanBpsMin(_minOutBpsBalToBptVal, minOutBpsBalToBptMin);
+        }
+
+        uint256 oldMinOutBpsBalToBptVal = minOutBpsBalToBptPtr.val;
+        minOutBpsBalToBptPtr.val = _minOutBpsBalToBptVal;
+
+        emit MinOutBpsBalToBptValUpdated(_minOutBpsBalToBptVal, oldMinOutBpsBalToBptVal);
+    }
+
+    ////////////////////////////////////////////////////////////////////////////
+    // PUBLIC: Owner
+    ////////////////////////////////////////////////////////////////////////////
+
+    function deposit(uint256 _amountBpt1, uint256 _amountBpt2) external onlyOwner {
+        if (_amountBpt1 == 0 && _amountBpt2 == 0) {
+            revert NothingToDeposit();
+        }
+
+        // TODO: See if can be moved elsewhere
+        uint256 bptDeposited1 = baseRewardPool1.balanceOf(address(this));
+        uint256 bptDeposited2 = baseRewardPool2.balanceOf(address(this));
+        if (bptDeposited1 == 0 && bptDeposited2 == 0) {
+            // Initialize at first deposit
+            lastClaimTimestamp = block.timestamp;
+        }
+
+        if (_amountBpt1 > 0) {
+            asset1.transferFrom(msg.sender, address(this), _amountBpt1);
+            AURA_BOOSTER.deposit(pid1, _amountBpt1, true);
+        }
+        if (_amountBpt2 > 0) {
+            asset2.transferFrom(msg.sender, address(this), _amountBpt2);
+            AURA_BOOSTER.deposit(pid2, _amountBpt2, true);
+        }
+
+        emit Deposit(address(asset1), _amountBpt1, block.timestamp);
```

Comment:
Wouldn't it make sense to add these events to the if statements above so that they only get emitted if a deposit actually went through for either of the pools?

Reply:
Good point

Reply:
Will push a fix